components/ActivatedNote.jsx

```diff
@@ -14,7 +14,7 @@ import './styles/icon.css';
  * @param {('small'|'medium'|'large')} [props.size=small] - the icon size
  * @param {string} [props.ariaLabel] - the label for accessibility
  */
-const ActivatedNote = ({size, ariaLabel, className, ...other}) => {
+const ActivatedNote = ({size = "medium", ariaLabel="", className, ...other}) => {
```

Comment:
I think ariaLabel should be written the same way as size. with spaces around the =